olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -559,6 +559,102 @@ spec:
                 - '*'
           serviceAccountName: knative-operator
     strategy: deployment
+  webhookdefinitions:
+    - generateName: validating.knativeeventings.operator.serverless.openshift.io
+      type: ValidatingAdmissionWebhook
+      deploymentName: knative-openshift
+      admissionReviewVersions:
+        - v1beta1
+      containerPort: 9876
+      failurePolicy: Ignore
+      rules:
+        - apiGroups:
+            - operator.knative.dev
+          apiVersions:
+            - v1alpha1
+          operations:
+            - CREATE
+            - UPDATE
+          resources:
+            - knativeeventings
+      sideEffects: None
+      webhookPath: /validate-knativeeventings
+    - generateName: validating.knativeservings.operator.serverless.openshift.io
+      type: ValidatingAdmissionWebhook
+      deploymentName: knative-openshift
+      admissionReviewVersions:
+        - v1beta1
+      containerPort: 9876
+      failurePolicy: Ignore
+      rules:
+        - apiGroups:
+            - operator.knative.dev
+          apiVersions:
+            - v1alpha1
+          operations:
+            - CREATE
+            - UPDATE
+          resources:
+            - knativeservings
+      sideEffects: None
+      webhookPath: /validate-knativeservings
+    - generateName: validating.knativekafkas.operator.serverless.openshift.io
+      type: ValidatingAdmissionWebhook
+      deploymentName: knative-openshift
+      admissionReviewVersions:
+        - v1beta1
+      containerPort: 9876
+      failurePolicy: Ignore
+      rules:
+        - apiGroups:
+            - operator.serverless.openshift.io
+          apiVersions:
+            - v1alpha1
+          operations:
+            - CREATE
+            - UPDATE
+          resources:
+            - knativekafkas
+      sideEffects: None
+      webhookPath: /validate-knativekafkas
+    - generateName: mutating.knativeeventings.operator.serverless.openshift.io
+      type: MutatingAdmissionWebhook
```

Comment:
do we need `knativekafkas`1 for mutatingadmissionwebhook?

Reply:
There's only a ValidatingWebhook for knativekafkas.